tests/checkup_test.go

```diff
@@ -250,18 +250,24 @@ func newRoleBinding(name, serviceAccountName, roleName string) *rbacv1.RoleBindi
 }
 
 func newConfigMap() *corev1.ConfigMap {
+	testConfig := map[string]string{
+		"spec.timeout":                                 "15m",
+		"spec.param.targetNode":                        "",
+		"spec.param.guestImageSourcePVCNamespace":      testNamespace,
+		"spec.param.guestImageSourcePVCName":           "centos-8-rt",
```

Comment:
off-topic: is it assumed that the guestImageSourcePVCName PVC is pre-installed on the cluster?

Reply:
After this PR there is no need to pass these parameters.
[Removed](https://github.com/kiagnose/kubevirt-realtime-checkup/compare/ffb8320c5db4824da4ad21ed427508675af53f1f..85160e6a89b2d0a3ceb4fa04549688ea4ede5be5) them in the last commit.